src/domain/mainwindow.py

```diff
@@ -465,4 +467,36 @@ def update_en_camera_list(self):
         for camera in cameras:
             if camera.enabled:
                 text = f"({camera.type.value}) {camera.id}"
-                self.ui.listWidget_en_cameras.addItem(text)
\ No newline at end of file
+                self.ui.listWidget_en_cameras.addItem(text)
+
+    def _init_logging_dropdown(self):
+        """Method to initialize logging levels in tooldbar dropdown"""
+
+        label = QLabel("Log level: ")
+        self.ui.toolBar.addWidget(label)
+
+        comboBox = QComboBox(self)
+        comboBox.setObjectName("logLevelComboBox")
+        levels = ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']
+        comboBox.addItems(levels)
+        comboBox.setToolTip("Select logging level")
+        comboBox.setCurrentText("INFO")
+        comboBox.currentIndexChanged.connect(self.change_logging_level)
+        self.ui.toolBar.addWidget(comboBox)
+
+    def change_logging_level(self, index):
+        """Method to modify UI logging level"""
+
+        level_mapping = {
+            0: logging.DEBUG,
+            1: logging.INFO,
+            2: logging.WARNING,
+            3: logging.ERROR,
+            4: logging.CRITICAL
+        }
```

Comment:
This can be a CONSTANT defined at the top of the module. It's also much more efficient, because you don't have to always build the dictionary each time that the method is called.